adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -267,7 +267,7 @@ class ADAMContext private (@transient val sc: SparkContext) extends Serializable
    * @tparam T The type of records to return
    * @return An RDD with records of the specified type
    */
-  private[rdd] def loadParquet[T](
+  def loadParquet[T](
```

Comment:
I had to make this public again for unit tests in package `o.b.a.projections`.  It also allows for loading user-defined schema (e.g. extensions to bdg-formats) from Avro-in-Parquet files.